.env.example

```diff
@@ -25,4 +28,4 @@ ACCOUNT_ADDRESS=
 PRIVATE_KEY=
 
 # An EVM private to define a default EOA for EVM related scripts
-EVM_PRIVATE_KEY=
+EVM_PRIVATE_KEY=0x7f109a9e3b0d8ecfba9cc23a3614433ce0fa7ddcc80f2a8f10b222179a5a80d6
```

Comment:
I took this private key from Ganache, from [here](https://github.com/trufflesuite/ganache), they have been exposed for quite some time, so I guess we can safely use them, and it will also add a default funded account that people can simply import in their tools.